src/test/resources/org/prebid/server/it/test-application.properties

```diff
@@ -455,6 +455,8 @@ adapters.telaria.enabled=true
 adapters.telaria.endpoint=http://localhost:8090/telaria-exchange/
 adapters.theadx.enabled=true
 adapters.theadx.endpoint=http://localhost:8090/theadx-exchange
+adapters.tradplus.enabled=true
+adapters.tradplus.endpoint=http://localhost:8090/tradplus-exchange
```

Comment:
please add the macros to the url to check them in the integration test

Reply:
done.